listener/listener.go

```diff
@@ -37,43 +46,106 @@ func configure() {
 
 	eventsReader = kafka.NewReader(eventsConfig)
 
+	inventoryAddr := utils.GetenvOrFail("INVENTORY_ADDRESS")
+
+	config := inventory.NewConfiguration()
+	config.Debug = true
+	config.BasePath = inventoryAddr + INVENTORY_API_PREFIX
+
+	inventoryClient = inventory.NewAPIClient(config)
+
+	cfg := vmaas.NewConfiguration()
+	cfg.BasePath = utils.GetenvOrFail("VMAAS_ADDRESS") + VMAAS_API_PREFIX
+	cfg.Debug = true
+
+	vmaasClient = vmaas.NewAPIClient(cfg)
```

Comment:
Could we replace `base/vmaas_client` with that?

Reply:
Well, that's the goal.

Reply:
Ok, so could you remove base/vmaas_client package. Maybe you can use vmaas mock for testing your client, if you want.